src/hotspots/HotspotMaterial.js

```diff
@@ -266,16 +292,117 @@ FORGE.HotspotMaterial.prototype._createTextureFromImage = function(image)
 {
     this._displayObject = image;
 
-    this.setTextureFrame(image.frame);
+    this._texture = new THREE.Texture();
+    this._texture.generateMipmaps = false;
+    this._texture.minFilter = THREE.LinearFilter;
```

Comment:
Ok, will find a way to remove aliasing on VR on another pull request if it's ok for the team! This needs a lot of testing on different screen / texture sizes.